app/talk/comment-report-form.cjsx

```diff
@@ -1,26 +1,44 @@
-React = require 'react'
+React = {findDOMNode} = require 'react'
 Feedback = require './mixins/feedback'
+talkClient = require '../api/talk'
 
 module?.exports = React.createClass
   displayName: 'TalkCommentReportForm'
 
   mixins: [Feedback]
 
+  propTypes:
+    comment: React.PropTypes.object
+    error: ''
```

Comment:
`''` isn't valid here.

Aside: do `propTypes` ever really buy us anything? I tend to only define them in super generic components. `getDefaultProps` seems more practical.